force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +124,44 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
+        Integer vowelCount = 0;
+
+        // Convert the string to lowercase to ignore case
+        str = str.toLowerCase();
+
+        // Define a set of vowels
+        Set<String> vowels = new Set<String>{'a', 'e', 'i', 'o', 'u'};
+
+        // Split the string into characters
+        List<String> chars = str.split('');
 
         /*
         for () {
-            // get individual characters from the string - hint use substring or split
+            // Get the current characters from the string - hint  use substring or split
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
         }
         */
+        // Iterate through each character in the string
+        for (Integer i = 0; i < str.length(); i++ ) {
+
+            // Get the current character
+            chars = str.split('');
+
```

Comment:
You can streamline this by just using str.substring(i, i+1) to get the current character like you've done on the following line